keda/templates/manager/deployment.yaml

```
@@ -222,8 +222,10 @@ spec:
       {{- end }}
       nodeSelector:
         kubernetes.io/os: linux
-      {{- with .Values.nodeSelector }}
-        {{- toYaml . | nindent 8 }}
+      {{- if .Values.operator.nodeSelector }}
+        {{- toYaml .Values.operator.nodeSelector | nindent 8 }}
+      {{- else if .Values.nodeSelector }}
+        {{- toYaml .Values.nodeSelector }}
```

what about something like
```suggestion
      {{- with .Values.operator.nodeSelector | default .Values.nodeSelector }}
        {{- toYaml . | nindent 8 }}
```
This applies the new value and it defaults to the previous one